shared/src/model/exercise/answer.js

```diff
@@ -14,7 +14,9 @@ export default class ExerciseAnswer extends BaseModel {
   @belongsTo({ model: 'exercise/question' }) question;
 
   @computed get validity() {
-    if (isEmpty(this.content_html)) {
+    if (
+      isEmpty(this.content_html) && !get(this, 'question.isOpenEnded')
```

Comment:
I don't think an empty answer is valid, instead the answers array should be empty?